delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphPluginModule.scala

```diff
@@ -55,8 +56,22 @@ class BlazegraphPluginModule(priority: Int) extends ModuleDef {
       HttpClient()(cfg.indexingClient, as.classicSystem, sc)
   }
 
+  make[BlazegraphSlowQueryStore].from { (xas: Transactors) =>
+    BlazegraphSlowQueryStore(
+      xas
+    )
+  }
+
+  make[BlazegraphSlowQueryLogger].from { (cfg: BlazegraphViewsConfig, store: BlazegraphSlowQueryStore) =>
+    new BlazegraphSlowQueryLoggerImpl(store, cfg.slowQueryThreshold)
```

Comment:
Add an apply method in `BlazegraphSlowQueryLogger` and make `BlazegraphSlowQueryLoggerImpl` private